whd/webhook.py

```diff
@@ -39,6 +44,12 @@ def __init__(
         self.dispatcher = GithubWebhookDispatcher(cfg_set=cfg_set, whd_cfg=whd_cfg)
 
     def post(self):
+        executor.submit(self._post)
+        # resource checking can take long, so we execute in an own thread and return
+        time.sleep(3)
```

Comment:
no.. just wait for the submitted task's result with a timeout; ignore the timeout-exception (or rather, print a warning..). that#s it